components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/dto/APIProductDTO.java

```diff
@@ -0,0 +1,292 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.wso2.carbon.apimgt.rest.api.publisher.dto.APIBusinessInformationDTO;
+
+import io.swagger.annotations.*;
+import com.fasterxml.jackson.annotation.*;
+
+import javax.validation.constraints.NotNull;
+
+
+
+
+
+@ApiModel(description = "")
+public class APIProductDTO  {
+  
+  
+  
+  private String id = null;
+  
+  @NotNull
+  private String name = null;
+  
+  
+  private String version = null;
+  
+  
+  private String description = null;
+  
+  
+  private String provider = null;
+  
+  public enum VisibilityEnum {
+     PUBLIC,  PRIVATE,  RESTRICTED,  CONTROLLED, 
+  };
+  
+  private VisibilityEnum visibility = null;
+  
+  
+  private List<String> visibleRoles = new ArrayList<String>();
+  
+  
+  private List<String> visibleTenants = new ArrayList<String>();
+  
+  
+  private List<String> tags = new ArrayList<String>();
+  
+  
+  private String thumbnailUri = null;
+  
+  @NotNull
+  private List<String> throttlingTier = new ArrayList<String>();
+  
+  
+  private String status = null;
+  
+  
+  private List<String> apis = new ArrayList<String>();
+  
+  
+  private APIBusinessInformationDTO businessInformation = null;
+  
+  public enum SubscriptionAvailabilityEnum {
+     current_tenant,  all_tenants,  specific_tenants, 
+  };
+  
+  private SubscriptionAvailabilityEnum subscriptionAvailability = null;
+  
+  
+  private List<String> subscriptionAvailableTenants = new ArrayList<String>();
+
+  
+  /**
```

Comment:
How about adding a meaningful comment instead empty comment ? (In all getters and setters)